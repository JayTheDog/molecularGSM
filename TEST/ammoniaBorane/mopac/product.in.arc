

                     SUMMARY OF PM6 CALCULATION, Site No: 14762

                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:32:15 2016
                                                       No. of days left =   0

           Empirical Formula: H10 N2 B2  =    14 atoms

PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -39.67756 KCAL/MOL =    -166.01092 KJ/MOL
          TOTAL ENERGY            =       -607.32933 EV
          ELECTRONIC ENERGY       =      -2204.75629 EV
          CORE-CORE REPULSION     =       1597.42696 EV
          GRADIENT NORM           =          0.53604
          DIPOLE                  =          4.66796 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =         13
          IONIZATION POTENTIAL    =         10.866878 EV
          HOMO LUMO ENERGIES (EV) =        -10.867  0.467
          MOLECULAR WEIGHT        =         59.712
          COSMO AREA              =        127.47 SQUARE ANGSTROMS
          COSMO VOLUME            =        108.48 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    11    H     6     5.88240
            H     8    H     5     4.23898
            H    14    H     9     2.74689
          SCF CALCULATIONS        =         26
          WALL-CLOCK TIME         =          0.125 SECONDS
          COMPUTATION TIME        =          0.121 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  N     0.77625814 +1  -1.81180103 +1   1.79099017 +1
  B     0.71671396 +1   1.22654933 +1  -0.93495637 +1
  B    -1.01412765 +1   0.53976500 +1  -1.28440200 +1
  N    -0.30167887 +1   0.65321315 +1   0.09909283 +1
  H     0.13491097 +1  -2.38671168 +1   2.31489649 +1
  H     1.49644884 +1  -1.50406947 +1   2.42593253 +1
  H     1.21558274 +1  -2.40326898 +1   1.10152081 +1
  H     0.86906382 +1   2.40877166 +1  -0.89590386 +1
  H     1.66192581 +1   0.53087569 +1  -1.14725328 +1
  H    -0.02724614 +1   1.06712123 +1  -2.07327699 +1
  H    -1.91284087 +1   1.30430983 +1  -1.45895745 +1
  H    -1.11248127 +1  -0.57114637 +1  -1.70713906 +1
  H    -0.04667189 +1  -0.22123440 +1   0.56542494 +1
  H    -0.69257657 +1   1.30780479 +1   0.76143089 +1
 
