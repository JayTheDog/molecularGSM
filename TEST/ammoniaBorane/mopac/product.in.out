 *******************************************************************************
 ** Site#: 14762        For non-commercial use only     Version 15.168L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.168L web: HTTP://OpenMOPAC.net         Days remaining:   0    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Dec  2 19:32:15 2016  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  NOSYM      - POINT-GROUP SYMMETRY SET TO C1
 *  T=         - A TIME OF     10.0 MINUTES REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.74101200  *  -1.69799200  *   1.69298900  *
     2       B          0.74135500  *   1.18284800  *  -0.89473600  *
     3       B         -1.01168700  *   0.49684500  *  -1.25464700  *
     4       N         -0.29113100  *   0.60530700  *   0.11544400  *
     5       H          0.08814500  *  -2.26971600  *   2.22339400  *
     6       H          1.46939000  *  -1.40462900  *   2.34015000  *
     7       H          1.18635000  *  -2.31501300  *   1.01737200  *
     8       H          0.94110300  *   2.35781300  *  -0.79127000  *
     9       H          1.65652800  *   0.44773400  *  -1.12749800  *
    10       H         -0.03973400  *   1.08793200  *  -1.98370800  *
    11       H         -1.93760600  *   1.24288900  *  -1.38996400  *
    12       H         -1.07261000  *  -0.61144200  *  -1.69748800  *
    13       H         -0.01413500  *  -0.24988100  *   0.61870400  *
    14       H         -0.68793500  *   1.28476900  *   0.75219700  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         N        0.7410   -1.6980    1.6930
     2         B        0.7414    1.1828   -0.8947
     3         B       -1.0117    0.4968   -1.2546
     4         N       -0.2911    0.6053    0.1154
     5         H        0.0881   -2.2697    2.2234
     6         H        1.4694   -1.4046    2.3401
     7         H        1.1864   -2.3150    1.0174
     8         H        0.9411    2.3578   -0.7913
     9         H        1.6565    0.4477   -1.1275
    10         H       -0.0397    1.0879   -1.9837
    11         H       -1.9376    1.2429   -1.3900
    12         H       -1.0726   -0.6114   -1.6975
    13         H       -0.0141   -0.2499    0.6187
    14         H       -0.6879    1.2848    0.7522

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: H10 N2 B2  =    14 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =   13

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    28.167 HEAT: -38.40958
 CYCLE:     2 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    30.177 HEAT: -38.75378
 CYCLE:     3 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    20.750 HEAT: -38.93968
 CYCLE:     4 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    11.277 HEAT: -39.11939
 CYCLE:     5 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    10.948 HEAT: -39.18686
 CYCLE:     6 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    12.450 HEAT: -39.22653
 CYCLE:     7 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    10.504 HEAT: -39.27119
 CYCLE:     8 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     8.216 HEAT: -39.31924
 CYCLE:     9 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     5.294 HEAT: -39.35777
 CYCLE:    10 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     7.128 HEAT: -39.40256
 CYCLE:    11 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     5.026 HEAT: -39.45116
 CYCLE:    12 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     5.279 HEAT: -39.50513
 CYCLE:    13 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     6.361 HEAT: -39.54080
 CYCLE:    14 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     6.188 HEAT: -39.58174
 CYCLE:    15 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.386 HEAT: -39.62551
 CYCLE:    16 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.991 HEAT: -39.64033
 CYCLE:    17 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.471 HEAT: -39.65085
 CYCLE:    18 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.335 HEAT: -39.65930
 CYCLE:    19 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.682 HEAT: -39.66719
 CYCLE:    20 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.218 HEAT: -39.67140
 CYCLE:    21 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.063 HEAT: -39.67490
 CYCLE:    22 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.024 HEAT: -39.67651
 CYCLE:    23 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     0.536 HEAT: -39.67756

     GRADIENT =  0.53604  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:32:15 2016
                                                       No. of days left =   0




          FINAL HEAT OF FORMATION =        -39.67756 KCAL/MOL =    -166.01092 KJ/MOL


          TOTAL ENERGY            =       -607.32933 EV
          ELECTRONIC ENERGY       =      -2204.75629 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =       1597.42696 EV
          COSMO AREA              =        127.47 SQUARE ANGSTROMS
          COSMO VOLUME            =        108.48 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.53604
          IONIZATION POTENTIAL    =         10.866878 EV
          HOMO LUMO ENERGIES (EV) =        -10.867  0.467
          NO. OF FILLED LEVELS    =         13
          MOLECULAR WEIGHT        =         59.712

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    11    H     6     5.88240
            H     8    H     5     4.23898
            H    14    H     9     2.74689


          SCF CALCULATIONS        =         26
 
          WALL-CLOCK TIME         =          0.125 SECONDS
          COMPUTATION TIME        =          0.120 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.77625814  *  -1.81180103  *   1.79099017  *
     2       B          0.71671396  *   1.22654933  *  -0.93495637  *
     3       B         -1.01412765  *   0.53976500  *  -1.28440200  *
     4       N         -0.30167887  *   0.65321315  *   0.09909283  *
     5       H          0.13491097  *  -2.38671168  *   2.31489649  *
     6       H          1.49644884  *  -1.50406947  *   2.42593253  *
     7       H          1.21558274  *  -2.40326898  *   1.10152081  *
     8       H          0.86906382  *   2.40877166  *  -0.89590386  *
     9       H          1.66192581  *   0.53087569  *  -1.14725328  *
    10       H         -0.02724614  *   1.06712123  *  -2.07327699  *
    11       H         -1.91284087  *   1.30430983  *  -1.45895745  *
    12       H         -1.11248127  *  -0.57114637  *  -1.70713906  *
    13       H         -0.04667189  *  -0.22123440  *   0.56542494  *
    14       H         -0.69257657  *   1.30780479  *   0.76143089  *

                             CARTESIAN COORDINATES

   1    N        0.776258143    -1.811801030     1.790990171
   2    B        0.716713960     1.226549327    -0.934956367
   3    B       -1.014127649     0.539764998    -1.284401998
   4    N       -0.301678868     0.653213154     0.099092833
   5    H        0.134910972    -2.386711679     2.314896494
   6    H        1.496448836    -1.504069465     2.425932533
   7    H        1.215582738    -2.403268982     1.101520813
   8    H        0.869063818     2.408771659    -0.895903859
   9    H        1.661925807     0.530875691    -1.147253283
  10    H       -0.027246135     1.067121234    -2.073276994
  11    H       -1.912840870     1.304309832    -1.458957446
  12    H       -1.112481269    -0.571146366    -1.707139061
  13    H       -0.046671886    -0.221234397     0.565424944
  14    H       -0.692576571     1.307804790     0.761430894


           Empirical Formula: H10 N2 B2  =    14 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -31.97876 -31.50382 -21.05389 -18.14914 -17.06267 -16.81111 -16.77613 -14.86987
 -12.18760 -11.51491 -11.19951 -10.90594 -10.86688   0.46686   1.81343   2.52420
   3.74489   4.17555   4.32951   5.02168   5.35433   5.58761   6.09295   6.15148
   6.15629   6.58795


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          N          -0.612513        5.6125     1.58735     4.02516
    2          B          -0.429000        3.4290     1.07717     2.35183
    3          B          -0.427969        3.4280     1.07747     2.35050
    4          N          -0.148280        5.1483     1.41627     3.73201
    5          H           0.206299        0.7937     0.79370
    6          H           0.206748        0.7933     0.79325
    7          H           0.209016        0.7910     0.79098
    8          H           0.095881        0.9041     0.90412
    9          H           0.095927        0.9041     0.90407
   10          H           0.115766        0.8842     0.88423
   11          H           0.096101        0.9039     0.90390
   12          H           0.096582        0.9034     0.90342
   13          H           0.268058        0.7319     0.73194
   14          H           0.227384        0.7726     0.77262
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.314    -1.615     3.036     3.453
 HYBRID         0.511    -0.932     0.788     1.323
 SUM            0.826    -2.547     3.824     4.668


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  N   1.58735   1.28614   1.46949   1.26953
    2  B   1.07717   0.73363   0.97223   0.64596
    3  B   1.07747   0.71635   0.96344   0.67072
    4  N   1.41627   1.32265   1.22860   1.18076
    5  H   0.79370
    6  H   0.79325
    7  H   0.79098
    8  H   0.90412
    9  H   0.90407
   10  H   0.88423
   11  H   0.90390
   12  H   0.90342
   13  H   0.73194
   14  H   0.77262

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.12 SECONDS

 == MOPAC DONE ==
